core/domain/exp_domain.py

```diff
@@ -79,6 +79,9 @@
 CMD_RENAME_STATE = 'rename_state'
 # This takes an additional 'state_name' parameter.
 CMD_DELETE_STATE = 'delete_state'
+# This takes addition 'state_name', 'content_id', 'language_code' and 'html'
```

Comment:
The below string says 'content_html' and 'translation_html' while here it says 'html'. Which is correct?